aztec/src/main/kotlin/org/wordpress/aztec/source/Format.kt

```diff
@@ -30,9 +32,10 @@ object Format {
         CleaningUtils.cleanNestedBoldTags(doc)
         if (isCalypsoFormat) {
             // remove empty span tags
-            doc.select("*")
-                    .filter { !it.hasText() && it.tagName() == "span" && it.childNodes().size == 0 }
-                    .forEach { it.remove() }
+            val select: Elements = doc.select("*")
+            select.filter { element: Element ->
```

Comment:
Minor (🔍): Why exposing this `select` local variable if not using it, maybe you had something in mind? 🤔 Consider, reverting it to it's previous form: `doc.select("*").filter { element: Element -> ... }`

Reply:
I thought it'll be more readable, but maybe it's not the case. Inlined in: 5bee1c97376322727875b624ae26048542e608ae